README.md

```diff
@@ -92,6 +94,31 @@ Or scanning for tests marked with the [Tests] attribute:
         return Tests.DefaultMainThisAssembly(args);
     }
 
+### Using Fuchu with Fable ###
+
+In order to be able to test your code both with .net and Fable you can adjust your main:
+
+    open Fuchu
+    #if FABLE_COMPILER
+    let exitIfNonZero v =
+        if v <> 0 then
+            failwithf "expected a nonzero exitcode, but got %i" v
+        v
+    #endif
+
+    [<EntryPoint>]
+    let main args =
+        defaultMain Tests.tests args
+        #if FABLE_COMPILER
+        |> exitIfNonZero
+        #endif
+
+Note that we don't use assembly scanning with Fable.
+
+In order to run the tests with Fable you then use `dotnet fable` to run your test project:
+
+    dotnet fable YourTestProject --outDir bin --runScript ./bin /d

```

Comment:
When I tested it, I think it also worked without passing the ./bin argument at the end, because runScript already loads the last file which will load the other files with tests. It maybe also worth noting that you need to install the esm package from npm.

Reply:
Yes, for sure! I'll add it to the section.